src/haddock/libs/libplots.py

```diff
@@ -138,8 +152,29 @@ def box_plots(capri_filename, cl_ranking):
             legend=dict(x=1.01, y=1.0, font_family="Helvetica", font_size=16),
             hoverlabel=dict(font_size=16, font_family="Helvetica"),
             )
-
         fig.write_html(px_fname, full_html=False, include_plotlyjs='cdn')
+        # create png boxplot if necessary
+        if png:
```

Comment:
Also why this check here?

Reply:
this checks if we have to create a matplotlib boxplot!

Reply:
I see... but can't you do the png with plotly? Check herer [`plotly.io.write_image`](https://plotly.github.io/plotly.py-docs/generated/plotly.io.write_image.html#plotly.io.write_image)

```python
if png:
    fig.write_image(file=px_fname, frormat="png")
else:
    fig.write_html(file=px_fname, full_html=False, include_plotlyjs='cdn')
```

Reply:
yep, I know, but the output is quite ugly..not a viable option if you want to create almost-publication-ready figures

not to mention the fact that the figure itself must be tweaked in order to take into account for the static nature of the png (downsizing of scatter points, etc..)..matplotlib is much better for png images overall

Reply:
Ah! If the point is to provide a figure that people can copy and paste it makes sense to make one that its actually useful for that purpose, all clear!